submissions/armenter66/popUp/index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>PopUp Armen</title>
+		<link rel="stylesheet" href="./style.css" />
+	</head>
+	<body>
+		<div class="container">
+			<header class="header">
+				<div class="header__wrapper">
+					<div class="header__logo">
+						<a href="./" class="header__link">
+							<img src="img/logo.jpg" alt="Avatar" class="header__logo-icon" />
+						</a>
+						<a href="./" class="header__title">
+							<h1>Armen Barsehian</h1>
+						</a>
+					</div>
+					<nav class="header__popup">
+						<div class="dropdown">
+							<label for="dropdown-1" class="dropdown__label">
+								<img
+									src="img/popup-icons/popup-button.png"
+									alt="PopUp Icon"
+									class="dropdown__icon"
+								/>
```

Comment:
Please, delete all alt attrtibutes values in popUP and nav icons (img tags). If image is used for decorations and don't have any sanse or information for user, alt attribute should be empty, without any symbols or spacing.
It should look like <img src="./image.png" alt=""/>
Fix this for all cases in nav and popUp.